openupgradelib/openupgrade.py

```diff
@@ -917,6 +918,104 @@ def rename_models(cr, model_spec):
     # TODO: signal where the model occurs in references to ir_model
 
 
+def merge_models(cr, old_model, new_model, ref_field):
+    """
+    Update model references for models that have merged to an existing model.
+    :param old_model: old model
+    :param new_model: destination model
+    :param ref_field: name of the field in new model that references old model.
```

Comment:
I would prefer if this mentioned that it refers to the `id` in the old model's table (if that is indeed the case).